manifests/base/sensor-controller/sensor-controller-deployment.yaml

```diff
@@ -19,6 +19,7 @@ spec:
       containers:
         - name: sensor-controller
           image: quay.io/argoproj/argo-events:latest
+          imagePullPolicy: Always
```

Comment:
This changes the default behaviour? Which was - `Always` if `latest` and `IfNotPresent` otherwise.

Reply:
When `imagePullPolicy` is missing, it defaults to `Always`.